xbmc/music/tags/MusicInfoTag.h

```diff
@@ -249,6 +253,8 @@ class CMusicInfoTag final : public IArchivable, public ISerializable, public ISo
   int m_samplerate;
   int m_channels;
   int m_bitrate;
+  std::string m_stationName;
+  std::string m_thumbURL;
```

Comment:
I see possible future confusion arising from this naming. Other items with a CMusicInfoTag (artists and albums) use thumbURL for a `CScraperUrl` property, which is in effect a list of URLs of availlable remote artwork. These values never get held in the CMusicInfoTag structure although many other properties do. Embedded art from music files is held in `EmbeddedArtInfo m_coverArt`.

For clarity I prefered `m_shoutcastCover` of the other PR because this property is only used for Shoutcast art, or maybe `m_coverShoutcast`

Reply:
Well, this feature is implemented completely generic - even in the original PR, although currently only shoutcast file fills the music tag's thumb URL. Renaming thumbURL which is generic to shoutcastSomething makes no real sense, sorry. I will do, if you insist, but if another non-shoutcast use case for this arises, we will need to rename that member... which can be avoided in the first place.

Reply:
@ksooo I would far rather reason it out than insist.
`m_thumbURL` is a bad choice of name. All music items can have thumbs, all of them except this Shoutcast facility store the path of the thumb along with other art elsewhere and not in `CMusicInfoTag`. There has been a consistent effort to clean up how an where artwork is stored, and this naming takes a backwards step and so I would prefer something else.

You see this as a generic addition, but the same could be said of station name. Why not call that  "extra name" and something in the future that wants to pass a string via `CMusicInfoTag` structure (that isn't covered  by title, album  or artist) could use it. It would be silly. The members of `CMusicInfoTag` are specific and not generic. I can not see anything in the future that would need to store an image URL in CMusicInfoTag, in fact I would only want that if there was no other way (as there isn't with Shoutcasts), but if "shoutcast" seems to specific for you then I suggest something like this
```
 std::string m_remoteArt; // Used to fetch thumb URL for Shoutcasts
```

Reply:
>  std::string m_stationArt; // Used to fetch thumb URL for Shoutcasts

 Yep, let's do it like this. Will change. Thanks.